### DISCUSSION TITLE
DatePicker defaultValue error: `Invalid ISO 8601 date string`

### DISCUSSION THREAD

**Discussion by cholazzzb** [chakra-ui/ark#2264] [Q&A] [answered]
Step to Reproduce:
1. Using DatePicker component like the example below:
```tsx
<DatePicker.Root defaultValue={['2024/12/31']}>
```

and get the `Invalid ISO 8601 date string` on the browser console.

What is the expected value for defaultValue props?

**Comment by segunadebayo** [comment, marked as answer]
Hi @cholazzzb,

The ISO format is YYYY-MM-DD

Learn more here https://www.iso.org/iso-8601-date-and-time-format.html
